lib/puppetfactory.rb

```diff
@@ -100,6 +100,7 @@ def settings.puppetfactory
   get '/new/:username' do |username|
     protected!
     create(username)
+    redirect '/'
```

Comment:
This is not the endpoint that should be called by the new user dialog. That should call the `/new` endpoint with posted data to set the username and password.

Reply:
@binford2k should we even have this endpoint then?

Reply:
No, but that time is not for today. This might be used by the CLI client. I plan some time after PC to simplify how all that works.